pkg/csi_driver/node.go

```diff
@@ -100,10 +106,17 @@ func (s *nodeServer) NodePublishVolume(ctx context.Context, req *csi.NodePublish
 	}
 	defer s.volumeLocks.Release(targetPath)
 
+	vs, ok := s.volumeStateStore[targetPath]
```

Comment:
for the two modes:
1. bucketName "_"
2. skipBucketAccessCheck=true

we do not need to bookkeep an entry in the map an optimize of the map mem usage. Thoughts?

Reply:
Done.